src/components/home_page/Home.jsx

```diff
@@ -137,41 +154,24 @@ const NorthAmerica = (props) => {
 }
 
 function Home() {
-  return (
-    <div className="home">
-      <title>Donate to Fight COVID-19</title>
-      <div class="grid">
-        <Box name="Global" list_of_links={GLOBAL} className="global" />
-        <Box name="Asia" list_of_links={ASIA} className="asia" />
-        <Box name="Europe" list_of_links={EUROPE} className="europe" />
-
-        <NorthAmerica
-        name="North America"
-        list_of_links={NORTH_AMERICA}
-        className="northamerica">
-        </NorthAmerica>
-
-        <Box
-          name="Central America"
-          list_of_links={CENTRAL_AMERICA}
-          className="centralamerica"
-        />
-        <Box
-          name="South America"
-          list_of_links={SOUTH_AMERICA}
-          className="southamerica"
-        />
-
-        <Box name="Africa" list_of_links={AFRICA} className="africa" />
-        <Box name="Oceania" list_of_links={OCEANIA} className="oceania" />
-        <Box
-          name="Middle East"
-          list_of_links={MIDDLE_EAST}
-          className="middleeast"
-        />
-      </div>
-    </div>
-  );
+	return (
+		<div className="home">
+			<title>Donate to Fight COVID-19</title>
+			<div class="grid">
+				<Box name="Global" list_of_links={GLOBAL} className="global" />
+				<Box name="Asia" list_of_links={ASIA} className="asia" />
+				<Box name="Europe" list_of_links={EUROPE} className="europe" />
+
+				<NorthAmerica name="North America" list_of_links={NORTH_AMERICA} className="northamerica" />
+				<Box name="Central America"	list_of_links={CENTRAL_AMERICA}	className="centralamerica"	/>
+				<Box name="South America"	list_of_links={SOUTH_AMERICA}	className="southamerica" />
+
+				<Box name="Africa" list_of_links={AFRICA} className="africa" />
+				<Box name="Oceania" list_of_links={OCEANIA} className="oceania" />
+				<Box name="Middle East"	list_of_links={MIDDLE_EAST}	className="middleeast"/>
+			</div>
+		</div>
+	);
 }
 
 export default Home;
```

Comment:
Please use the `prettier` extension on Visual Studio Code and lint the code so it complies with formatting standards - and so you're not changing all the code from 2 spaces to tabs every single time you edit a file!